### ISSUE TITLE
LegacyAuthProtocolsEnabled false

### ISSUE THREAD

**Issue by mores** [MicrosoftDocs/office-docs-powershell#540]
How should one connect when 
Set-SPOTenant -LegacyAuthProtocolsEnabled $False

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3b3240d3-6a7d-fc4b-62fd-d3a5375d8a4a
* Version Independent ID: 561212df-4dab-92b7-b634-57d53d313c6f
* Content: [PnP PowerShell overview](https://docs.microsoft.com/en-us/powershell/sharepoint/sharepoint-pnp/sharepoint-pnp-cmdlets?view=sharepoint-ps)
* Content Source: [sharepoint/docs-conceptual/sharepoint-pnp/sharepoint-pnp-cmdlets.md](https://github.com/MicrosoftDocs/office-docs-powershell/blob/master/sharepoint/docs-conceptual/sharepoint-pnp/sharepoint-pnp-cmdlets.md)
* Service: **unspecified**
* GitHub Login: @Techwriter40
* Microsoft Alias: **kirks**

**Comment by Techwriter40** [comment]
@VesaJuvonen one more PnP cmdlet comment for you .

**Comment by AndreaBarr** [comment]
@mores Thank you for this feedback

I am going to get this over to the PowerShell team so we can get this question answered for you.

I was able to get this into the backlog for the PowerShell team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated. 
 
I made a note to request the team to update this when the work is complete.
